### DISCUSSION TITLE
Add sort to Bookmarks

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16310] [Feature Requests]
Bookmarks of a collection come in the UI as a sub-menu of the collection which is very handy.
However sorted only alphabetically.

This is not ideal when the different filters are wanted sorted according to different stages.
OF course, it is possible to set a prefix (A-.. B-.. etc. or 01-...  02-.. etc.) but it would be nicer to be able to set a `sort` field.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
